README.rst

```diff
@@ -130,6 +130,8 @@ Detailed `installation instructions`_ are available.
 
 3. Install the `vagrant-service-manager`_ Vagrant plugin:
 
+   ``sudo dnf install libvirt-devel``
```

Comment:
@hferentschik You are right. If I use VirtualBox, I should need libvirt. However, when I install the plugin it fails to install because the libvirt-devel is not installed. How would the plugin install know that I won't use libvirt at all. I think it has something to do with the dependencies. 

For now this fixes my problem.